com.microsoft.mrtk.input/Simulation/Devices/SimulatedController.cs

```diff
@@ -55,6 +55,7 @@ internal class SimulatedController : IDisposable
     {
         private readonly MRTKSimulatedController simulatedController = null;
         private readonly IHandRay handRay = new HandRay();
+        private ControllerSimulationSettings controllerSimulationSettings;
```

Comment:
nit: do we want this to be `readonly` like the two fields above it?

Reply:
Done